src/courseware/course/sidebar/Sidebar.jsx

```diff
@@ -1,15 +1,34 @@
-import React from 'react';
+import React, { useContext } from 'react';
```

Comment:
[nit]: Do we need to import the entire react library in this file?

Reply:
Removed